### PR TITLE
working command line

### PR DESCRIPTION
**Pull request by dpensi**
# Description

locust implementation to perform load testing in several environments 

Fixes CCXDEV-7770

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## Testing steps

follow the new chapter in the `README` file : "Load Testing"

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+RUN dnf -y --setopt=tsflags=nodocs install zlib python3.8 git wget gcc make python3-devel && \
+    wget https://password.corp.redhat.com/RH-IT-Root-CA.crt && \
+    cp RH-IT-Root-CA.crt /etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.crt && \
+    update-ca-trust
+
+RUN python3 -m pip install --upgrade pip && \
+    pip3 install .
+
+USER 1001
```
Comment (joselsegura) [inline Dockerfile:18]: If the Dockerfile is intended only for using the Python tool, I would put it in a different folder. Being in the root directory of the repo can be misleading

File: setup.cfg [hunk 2]
```diff
@@ -3,3 +3,41 @@ max-line-length = 100
...
+
+[options.packages.find]
+exclude =
+    docs*
+
+[options.entry_points]
+console_scripts =
+    st = stage_tester.st:main
+    pta = stage_tester.pta:main
+    st_xlsx = stage_tester.st_xlsx:main
```
Comment (joselsegura) [inline setup.cfg:43]: Love it :-)

**Review by joselsegura — APPROVED** [review]
Probably I will move both tools to a different directory together and put there the setuptools and Dockerfile files, but apart of the nitpicks, LGTM

File: README.md [hunk 3]
```diff
@@ -1183,6 +1188,53 @@ pta.py -i times.csv -v -b 100
...
+
+first you need to install, from the project root launch
+
+```
+pip install .
+```
+
+this will install locust along all the packages, and will make available from the command line
+tools like `st`, `pta` and `st_xls`.
+Than we need to set up some environment variables or edit the file `config_dev.toml`. The
```
Comment (juandspy) [inline README.md:1209]: Than -> Then

**Review by juandspy — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (dpensi) [reply Dockerfile:18]: python tooling is everywhere except the `anim` folder, I'll leave it as is for now

**Pull request merged**